react/src/components/ResourceAllocationFormItems.tsx

```diff
@@ -776,77 +747,125 @@ const ResourceAllocationFormItems: React.FC<
                           },
                           {
                             type: 'number',
-                            min: sliderMinMax[currentAcceleratorType]?.min || 0,
+                            min:
+                              sliderMinMaxLimit[currentAcceleratorType]?.min ||
+                              0,
                           },
                           remainingValidationRules[currentAcceleratorType],
                         ]}
-                      />
+                      >
+                        <InputNumberWithSlider
```

Comment:
And I think we need to disable the AI accelerator input field when image does not support AI accelerator (e.g. simple python kernel).